src/Sylius/Component/Product/Model/VariantInterface.php

```diff
@@ -42,4 +42,14 @@ public function getAvailableOn();
      * @param null|\DateTime $availableOn
      */
     public function setAvailableOn(\DateTime $availableOn = null);
+
+    /**
+     * @return \DateTime
+     */
+    public function getAvailableUntil();
+
+    /**
+     * @param null|\DateTime $availableUntil
+     */
+    public function setAvailableUntil(\DateTime $availableUntil = null);
```

Comment:
I think it's not right to have default value of null here!? When are you calling it with no arguments? If you want to set to null you just `set(null)`